selfdrive/manager.py

```diff
@@ -479,7 +479,7 @@ def manager_thread():
     should_start = should_start and avail > 0.02
 
     # require usb power
-    should_start = should_start and msg.thermal.usbOnline
+    #should_start = should_start and msg.thermal.usbOnline #Test fixing OP3T bug.
```

Comment:
Remove this?

Reply:
Whoops... that was clearly on the wrong branch :)